Added list of letters

dahah-babah opened the pull request:
Added ability to delete letters
Correct time display added
Added checkboxes for letters

dahahbabah committed:
Added list of letters
Added ability to delete letters
Correct time display added
Added checkboxes for letters

ValentinBodrov commented on assets/js/appVue.js:
```diff
@@ -0,0 +1,171 @@
+        ],
+        "folder": "INBOX"
+    }
+];
```
Have u enter all this data manually? Where is it from?

dahah-babah replied:
This data was given to me by Max, because there is no way to get them by the request.

ValentinBodrov replied:
Ok thx

ValentinBodrov reviewed (commented):
There's a nice job, but I have a question:

ValentinBodrov reviewed (dismissed):
I approve this nice work

IReallyHateLogins commented:
@dahahORbabah,  will you add your other changes to this branch or should I merge right now?

IReallyHateLogins reviewed (commented):
Can you add a function witch would refresh your letters list, to then easily wire it up with the request to API instead of a stub we are using now? Also might be useful to connect his function to a button for now.

dahahORbabah commented:
> Can you add a function witch would refresh your letters list, to then easily wire it up with the request to API instead of a stub we are using now? Also might be useful to connect his function to a button for now.

I didn't quite understand your request. What should it refresh? I can add the button and the function.

dahahbabah committed:
Added a form for creating a new letter
Changed the function of deleting letters
Added some styles for buttons and letter items

IReallyHateLogins commented:
> > Can you add a function witch would refresh your letters list, to then easily wire it up with the request to API instead of a stub we are using now? Also might be useful to connect his function to a button for now.
> 
> I didn't quite understand your request. What should it refresh? I can add the button and the function.

Basically, i want you to put most of your global code in appVue.js inside a function, and that function can act as a stub for now. Later I'll replace this stub with request  to server.

IReallyHateLogins commented on assets/js/appVue.js:
```diff
@@ -0,0 +1,219 @@
+
+for (let i = 0; i < letterInJSON.length; i++) { 
+    letters.message.push(letterInJSON[i]);        
+}
```
I think we should wrap this into function, that simulates request to the server. It should first clear message list and, for now, populate it like this. We can connect it to a refresh button, and also call it when page is loaded. Later I can just add request handling to it instead of a stub, and page will function right away as it should.

dahah-babah replied:
Ok.

IReallyHateLogins reviewed (commented):
Seems great, I have just one suggestion. But we can merge as is.

dahahORbabah commented:
> > > Can you add a function witch would refresh your letters list, to then easily wire it up with the request to API instead of a stub we are using now? Also might be useful to connect his function to a button for now.
> > 
> > 
> > I didn't quite understand your request. What should it refresh? I can add the button and the function.
> 
> Basically, i want you to put most of your global code in appVue.js inside a function, and that function can act as a stub for now. Later I'll replace this stub with request to server.

Ok.

ValentinBodrov reviewed (commented):
Take away all the comments, and we'll approve it successfully

dahahbabah committed:
Added function for updating letters list

IReallyHateLogins reviewed (approved):
Well done!
